packages/cli/src/oclif/commands/jobs.js

```diff
@@ -0,0 +1,81 @@
+const { chain } = require('lodash');
+
+const BaseCommand = require('../ZapierBaseCommand');
+const { listMigrations } = require('../../utils/api');
+const { buildFlags } = require('../buildFlags');
+
+const getVersion = (versionStr) => versionStr.split('@')[1];
+
+class JobsCommand extends BaseCommand {
+  async perform() {
+    /**
+     * Migrations and Jobs are used somewhat interchangeably here.
+     * Migrations represents the collection of promotion and migration
+     * background "jobs" that were recently executed on this app.
+     */
+
+    this.startSpinner('Loading jobs');
+    const { migrations } = await listMigrations();
+    this.stopSpinner();
+
+    const jobs = chain(migrations)
+      .filter(
+        (migration) =>
+          migration.job_kind === 'migrate' || migration.job_kind === 'promote'
+      )
+      .map((migration) => ({
+        app_title: migration.app_title,
+        job_id: migration.job_id,
+        job_kind: migration.job_kind,
+        job_stage: migration.job_stage,
+        version_from: getVersion(migration.from_selected_api),
+        version_to: getVersion(migration.to_selected_api),
+        current_step: migration.progress.current_step.name,
+        current_progress: `${migration.progress.current_step.finished_points} finished, ${migration.progress.current_step.skipped_points} skipped, ${migration.progress.current_step.estimated_points} estimated`,
+        current_step_status: migration.progress.current_step.status,
+        overall_progress:
+          parseFloat(migration.progress.overall_progress * 100).toFixed(2) +
+          '%',
+        updated_at: migration.updated_at,
+        error_message: migration.error ? migration.error.message : '-',
+      }))
+      .sortBy((migration) => migration.updated_at, 'asc')
```

Comment:
[`sortBy()`](https://lodash.com/docs/#sortBy) doesn't seem to have an option for `asc` or `desc`. [`orderBy()`](https://lodash.com/docs/#orderBy) does. But since `asc` is the default, we can omit it.

Also, I think sorting by `started_at` makes it more intuitive:

```suggestion
      .sortBy((migration) => migration.started_at)
```